wal/decoder.go

```diff
@@ -98,6 +113,44 @@ func (d *decoder) decodeRecord(rec *walpb.Record) error {
 	return nil
 }
 
+// isTornEntry determines whether the last entry of the WAL was partially written
+// and corrupted because of a torn write.
+func (d *decoder) isTornEntry(data []byte) bool {
```

Comment:
I do not quite understand the example. There are two `A`s. Can you make it a little bit more clear?